### DISCUSSION TITLE
Atualizar versão 4.24

### DISCUSSION THREAD

**Discussion by Ricardo-2202** [RamonSilva20/mapos#1868] [General]
Resolvi atualizar a versão 4,24 fiz o backup e atualizei o Mapos. 
No entanto quando restauro o Backup o sistema para de funcionar.
Alguem já passou por isto e pode ajudar?

**Comment by Henrique-Miranda** [comment]
Já tentou restaurar o banco diretamente pelo phpmyadmin? Lá você saberia se está ocorrendo algum erro na importação do backup
